tests/run-make/incr-prev-body-beyond-eof/rmake.rs

```diff
@@ -0,0 +1,28 @@
+// After modifying the span of a function, if the length of
+// the span remained the same but the end line number became different,
+// this would cause an internal compiler error (ICE), fixed in #76256.
+
+// This test compiles main.rs twice, first with end line 16 and
+// then with end line 12. If compilation is successful, the end line
+// was hashed by rustc in addition to the span length, and the fix still
+// works.
+
+use run_make_support::{rustc, target, tmp_dir};
+use std::fs;
+
+fn main() {
+    fs::create_dir(tmp_dir().join("src"));
+    fs::create_dir(tmp_dir().join("incr"));
```

Comment:
These operations (creating directories, copying etc.) will be repeated in many tests. I would create utility functions for them in `run-make-support`, so that they can do `.unwrap()` inside, to avoid repeating it everywhere.